pkg/quota/image/imagestreamtag_evaluator.go

```diff
@@ -49,87 +49,58 @@ func NewImageStreamTagEvaluator(osClient osclient.Interface) kquota.Evaluator {
 		return obj, nil
 	}
 
-	return quotautil.NewSharedContextEvaluator(
-		imageStreamTagEvaluatorName,
-		kapi.Kind("ImageStreamTag"),
-		map[admission.Operation][]kapi.ResourceName{admission.Update: computeResources},
-		computeResources,
-		matchesScopeFunc,
-		getFuncByNamespace,
-		nil,
-		imageStreamTagConstraintsFunc,
-		makeImageStreamTagUsageComputerFactory(osClient))
+	return &generic.GenericEvaluator{
+		Name:                       imageStreamTagEvaluatorName,
+		InternalGroupKind:          imageapi.Kind("ImageStreamTag"),
+		InternalOperationResources: map[admission.Operation][]kapi.ResourceName{admission.Update: computeResources},
```

Comment:
`IST` virtual resource doesn't have `Create` method. It may be a case in the future with @deads2k's #8497. Until then, let's safe some CPU cycles during evaluation.